package.json

```diff
@@ -37,6 +37,7 @@
 		"@types/pg": "7.4.3",
 		"@types/request": "2.47.0",
 		"@types/session-file-store": "1.2.0",
+		"body-parser": "^1.18.2",
```

Comment:
Did you mean to add this to `devDependencies` ?

Reply:
Also, explicit versions please, i.e. remove the `^`.

Reply:
removed